mods/d2k/rules/defaults.yaml

```diff
@@ -337,6 +341,8 @@
 ^Plane:
 	Inherits@1: ^ExistsInWorld
 	Inherits@2: ^SpriteActor
+	UpdatesPlayerStatistics:
+		AddToArmyValue: true
```

Comment:
This will include the reinforcement carryall that delivers harvesters when a refinery is placed, the frigate that delivers starport units, and the ornithopter airstrike aircraft - these are all uncontrollable support actors and definitely shouldn't count - and also player-built carryalls, which could be argued either way.

Reply:
Dropped.